kfs-purap/src/main/resources/edu/arizona/kfs/workflow/PaymentRequestDocument.xml

```diff
@@ -0,0 +1,131 @@
+            <branch name="True">
+              <role name="Tax" nextNode="JoinVendorIsEmployeeOrNonResidentAlien" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOpVendorIsEmployeeOrNonResidentAlien" nextNode="JoinVendorIsEmployeeOrNonResidentAlien" />
+            </branch>
+            <join name="JoinVendorIsEmployeeOrNonResidentAlien" />
+          </split>
+          <split name="RequiresAutoApprovalNotification">
+            <branch name="True">
+              <role name="AutoApprove" nextNode="joinOnAutoApproval" />
+            </branch>
+            <branch name="False">
+              <simple name="NoOpAutoApproval" nextNode="joinOnAutoApproval" />
+            </branch>
+            <join name="joinOnAutoApproval" />
+          </split>
+        </routePath>
+      </routePaths>
+      <routeNodes>
+        <start name="AdHoc" />
+        <split name="RequiresImageAttachment">
+          <type>org.kuali.kfs.sys.document.workflow.SimpleBooleanSplitNode</type>
+        </split>
+        <role name="ImageAttachment">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <join name="JoinRequiresImageAttachment" />
+        <simple name="NoOpRequiresImageAttachment">
+          <type>org.kuali.rice.kew.engine.node.NoOpNode</type>
+        </simple>
+        <split name="PurchaseWasReceived">
+          <type>org.kuali.kfs.sys.document.workflow.SimpleBooleanSplitNode</type>
+        </split>
+        <role name="Receiving">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <join name="JoinReceiving" />
+        <simple name="NoOpReceiving">
+          <type>org.kuali.rice.kew.engine.node.NoOpNode</type>
+        </simple>
+        <role name="SubAccount">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="Account">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+          <mandatoryRoute>true</mandatoryRoute>
+        </role>
+        <role name="AccountingOrganizationHierarchy">
+          <qualifierResolverClass>org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
+          <activationType>P</activationType>
+        </role>
+        <role name="ObjectSubTypeCode">
+          <qualifierResolverClass>org.kuali.rice.kns.workflow.attribute.DataDictionaryQualifierResolver</qualifierResolverClass>
```

Comment:
should be:
org.kuali.rice.krad.workflow.attribute.DataDictionaryQualifierResolver